modules/tinymce/src/plugins/table/main/ts/actions/ResizeHandler.ts

```diff
@@ -52,6 +54,12 @@ export const getResizeHandler = function (editor: Editor): ResizeHandler {
     // so check to see if it ends with `e` (eg east edge)
     const isRightEdgeResize = Strings.endsWith(origin, 'e');
 
+    // Responsive tables don't have a width so we need to convert it to a relative/percent
```

Comment:
I moved this, as this logic only applies to using the corner resizers. When using the bar resizers we need to do it on the resize start, not at the end.